### PR TITLE
Adding title and id in campaign match endpoint

### PR DESCRIPTION
**Pull request by spaceo**
#### Link to issue

https://reload.atlassian.net/browse/DDFSOEG-317

#### Description

We need title for statistics right now.
Adding id also because we might need that in the future too.

#### Checklist

- [x] My complies with [our coding guidelines](../documentation/code-guidelines.md).
- [x] My code passes our static analysis suite. If not then I have added a comment explaining why this change should be exempt from the code standards and process.
- [x] My code passes our continuous integration process. If not then I have added a comment explaining why this change should be exempt from the code standards and process.

#### Additional comments or questions

If you have any further comments or questions for the reviewer them please add them here.

### CODE REVIEW

**Review by kasperg — CHANGES_REQUESTED** [review]
CI checks are failing.

- Remember to update the specification in VCS
- Remember to Cypress tests accordingly

**Review by kasperg — APPROVED** [review]
:+1: LGTM

File: .github/workflows/ci-tests.yml [hunk 1]
```diff
@@ -222,4 +222,4 @@ jobs:
       - name: Download current specification
         run: task ci:openapi:download
       - name: Ensure specification has not drifted
-        run: git diff --exit-code openapi.json
+        run: git diff --ignore-space-at-eol --exit-code openapi.json
```
Comment (cableman) [inline .github/workflows/ci-tests.yml:225]: Why do we need to `ignore spaces at end of line` when comparing api spec. It should be generated by the same code and thereby be identical?

**Review by cableman — APPROVED** [review]
Approved with comment

File: .github/workflows/ci-tests.yml (hunk 1 above)
Comment (kasperg) [reply .github/workflows/ci-tests.yml:225]: We tried to explain in the commit comment adding the change:

> We'll allow spaces in eol to ignore diffs based on editor cleanup settings.

**Pull request merged**